maestro-cli/src/main/java/maestro/cli/session/MaestroSessionManager.kt

```diff
@@ -309,13 +308,18 @@ object MaestroSessionManager {
             logger = IOSDriverLogger(),
             deviceId = device.instanceId,
         )
-        val xcTestDriverClient = XCTestDriverClient(defaultHost, xcTestPort, isStudio) {
-            if (SessionStore.activeSessions().isNotEmpty()) {
-                IdbInstaller.setup(device)
-                return@XCTestDriverClient xcTestInstaller.setup()
+        val xcTestDriverClient = XCTestDriverClient(
+            host = defaultHost,
+            port = xcTestPort,
+            installNetworkInterceptor = isStudio,
+            restoreConnection = {
+                if (SessionStore.activeSessions().isNotEmpty()) {
+//                    IdbInstaller.setup(device)
```

Comment:
I think this is needed since the original java process stops which started both xctest runner and idb.

Reply:
What is the problem we are facing with idb starting here?